src/main/java/com/example/cafeinformation/CafeController.java

```diff
@@ -15,8 +16,7 @@ public CafeController(CafeMapper cafeMapper) {
     }
 
     @GetMapping("/cafes")
-    public List<Cafe> getNames() {
-        List<Cafe> cafes = cafeMapper.findAll();
-        return cafes;
+    public List<Cafe> findByPlaces(@RequestParam String equalsWith) {
```

Comment:
equalWithだと何とイコールなのかわからないので、
```suggestion
    public List<Cafe> findByPlaces(@RequestParam String place) {
```
でよいかと！